src/models/values.rs

```diff
@@ -56,6 +56,13 @@ macro_rules! decimal_scalar {
                 self.0
             }
 
+            pub fn to_f64(&self) -> f64 {
+                // The rust_decimal crate unwraps like this when implementing serde-with-float.
```

Comment:
so why are we not using the decimal crate exposed functionality instead of pulling in our own num-traits?

Reply:
because I can't pass the serializer to the json! macro... so I dug into their seralizer code and just took this snippet